conf/telemetry/highspeed_rotorcraft.xml

```diff
@@ -38,17 +39,22 @@
       <message name="INS_EKF2_EXT"             period="4.25"/>
       <message name="FBW_STATUS"               period="2.1"/>
       <message name="WIND_INFO_RET"            period="9.2"/>
-      <message name="AHRS_BIAS"                period="7.5"/>
       <message name="HOVER_LOOP"               period="0.3"/>
       <message name="GUIDANCE_H_REF_INT"       period="0.31"/>
       <message name="GUIDANCE_INDI_HYBRID"     period="0.4"/>
       <message name="HYBRID_GUIDANCE"          period="0.4"/>
       <message name="ESC"                      period="0.5"/>
       <message name="AHRS_REF_QUAT"            period="0.1"/>
-      <message name="INS_FLOW_INFO"            period="0.1"/>
+      <message name="INS_FLOW_INFO"	           period="0.1"/> 
```

Comment:
Please keep the spacing consistent such that it looks organized.
Also, are all these modifications necessary?

Reply:
these changes are actually not needed, i pulled them back